napari/_qt/dialogs/qt_notification.py

```diff
@@ -358,38 +358,7 @@ def from_notification(
         if isinstance(notification, ErrorNotification):
 
             def show_tb(parent_):
```

Comment:
Not directly related to these changes, but should the typing of `ActionSequence` be updated that each action callable associated with a `NapariQtNotification` passes through the parent (i.e. a `QWidget`) as the only argument?

I bring this up because I was confused about how the parent actually gets passed through. Is my understanding correct? That is occurs when the action callable is executed? And the parent is the same as the parent of the original notification (likely mainwindow or some main widget in napari)?

Reply:
I have fixed `ActionSequence` definition. I'm not sure what the general idea for this mechanism was introduced by @Carreau.  

My first idea is that it is introduced to manipulate `NapariQtNotification` but it is closed by the same event. So maybe it should just pass the parent object.

Reply:
Might be a good idea to rename `parent_`? If I followed the existing logic properly here (and it's possible that I didn't!), then this will always be an instance of `NapariQtNotification`. In some sense that's the parent dialog of the dialog created here, but not in the sense of the `QObject` parent/child hierarchy.

Reply:
Done